src/Accounting/index.ts

```diff
@@ -0,0 +1,202 @@
+import { FlatOrder } from "../Types/accounting";
+
+const RYAN_6 = 6; // a number accredited to our good friend Ryan Garner
+const LIQUIDATION_GAS_COST = 25; // When gas price is 250 gwei and eth price is 1700, the liquidation gas cost is 25 USD.
```

Comment:
Eventually we will need a way for this to be way more dynamic. Could be cool for it to actually pull the gas price as reported by the Chainlink feed and also the ETH/USD price reported by teh Chainlink feed and run the calc.

Happy with this as a starting point though.